bin/deploy-vhost

```diff
@@ -440,21 +450,53 @@ sub create_or_update_files {
     make_symlink($web_dir, "$vhost_dir/docs");
     shell("chown", "-h", "$conf->{user_uid}:$conf->{user_gid}", "$vhost_dir/docs"); # perl's chown doesn't do symlinks
 
-    # Initial clone, for git repository that don't exist
-    my $git_url_base;
-    if (exists($conf->{git_repository})) {
-        if (exists($conf->{git_url_base})) {
-            $git_url_base = $conf->{git_url_base};
-        } elsif ($conf->{git_user} eq 'anon') {
-            $git_url_base = "git://git.mysociety.org/"
-        } else {
-            $git_url_base = "ssh://" . $conf->{git_user} . '@git.mysociety.org/data/git/public/';
+    if ($from_package) {
+        print "Will deploy from package $vhost_package.";
+        print "Fetching $vhost_package...";
+        shell(
+            "curl",
+            "https://msdevpkg003cama.srv.mysociety.org:49090/$vhost_package",
```

Comment:
Let's use a CNAME here so we don't need to hardcode a potentially transient instance hostname.